source/manual/deploy-fixes-for-a-security-vulnerability.html.md

```diff
@@ -18,7 +18,39 @@ This repository should be up to date as of the previous release, but will be
 missing any unreleased commits that are on GitHub master.
 This is fine as you don't want to deploy these.
 
-## Deploy process
+## Applying the fix
+
+1. In the root of the local repo, run the following commands to install the AWS
+   credential helper and add CodeCommit as a remote:
+   
+   `git config credential.helper '!aws codecommit credential-helper $@'`
+   `git config credential.UseHttpPath true`
+   `git remote add aws https://git-codecommit.eu-west-2.amazonaws.com/v1/repos/<app>`
+
+1. [Get some AWS credentials](/manual/deploying-terraform.html#2-get-your-credentials)
```

Comment:
Is it worth mentioning that you need to use your govuk-tools arn?

Reply:
Fixed